storybook/storybook-react/src/components/Heading/Heading.stories.tsx

```diff
@@ -11,21 +11,51 @@ const meta = {
   component: Heading,
   args: {
     children: 'Jouw typograaf biedt mij zulke exquise schreven!',
-    level: 1,
-    size: 'level-1',
+    level: undefined,
+    size: undefined,
+    inverseColor: false,
   },
   argTypes: {
+    level: {
+      control: {
+        type: 'number',
+        min: 1,
+        max: 4,
+      },
+    },
     size: {
+      control: 'radio',
       options: ['level-1', 'level-2', 'level-3', 'level-4'],
     },
+    inverseColor: { control: 'boolean' },
   },
 } satisfies Meta<typeof Heading>
 
 export default meta
 
 type Story = StoryObj<typeof meta>
 
-export const Heading1: Story = {}
+export const Default: Story = {
+  decorators: [
+    (Story, context) => (
+      <div
+        style={{
+          background: context.args.inverseColor ? '#004699' : 'transparent',
+          padding: context.args.inverseColor ? '16px' : 0,
+        }}
+      >
+        <Story />
+      </div>
+    ),
+  ],
+}
+
+export const Heading1: Story = {
+  args: {
+    level: 1,
+    size: 'level-1',
```

Comment:
I think we shouldn’t specify the size if it’s the same as the level.

And maybe even remove level for heading 1?

Reply:
Hm, I chose this approach so you would have code examples of all levels and all sizes, even though it's not necessary to specify the size when it's the same as the level. But I can see why this would be confusing.

The verbose approach would be to show all possible sizes for every level, but that would be a bit much. 

We could show the four sizes in order with different levels? Or, if we just want a visual example of the four sizes,  we could show the four levels in order?

Reply:
Yeah, what you expect is what I tried to describe with my second option 😄 I'll do that for now.

I guess the approach depends on whether you see the examples as something developers can quickly copy-paste, or just visual examples. But, you're right, that's for a different PR

Reply:
👍🏼 Developers can quickly copy-paste after clicking ‘Show code’. That will happen a couple of times per month. Viewing hopefully a hundredfold more.